coding-concepts.js

```diff
@@ -19,26 +19,30 @@ const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
 // a) Your answer:
+// 10 
 // b) Verify and explain:
-
+// Because .lenth returns the total number of elements by index and it's the last index plus 1.
 
```

Comment:
just to add some clarity. .length will count all the values in the array or string. Values in an array are separated by commas. Values in a string are all the characters including spaces within a set of quotation marks.